### DISCUSSION TITLE
Change build names for CPU-Mac version

### DISCUSSION THREAD

**Discussion by quy-ng** [pyg-team/pytorch_geometric#7788] [Ideas]
Recently, I found a problem when I tried to install Torch Geometric (CPU) with Poetry in MacOS but got an issue. I linked the issue for your reference https://github.com/orgs/python-poetry/discussions/8217 

I found that [https://data.pyg.org/whl/torch-2.0.0+cpu.html](https://data.pyg.org/whl/torch-2.0.0+cpu.html), packages for MacOS is missing `cpu` part. For example, 

`pyg_lib` CPU for Linux  `pyg_lib-0.2.0+pt20cpu-cp310-cp310-linux`
`pyg_lib` CPU for Mac `pyg_lib-0.2.0+pt20-cp310-cp310-macosx`

This leads Poetry to find `pyg_lib-0.2.0+pt20cpu` for Mac, but this is unavailable.

Could pyg team keep file naming consistent? I expect `pyg_lib-0.2.0+pt20cpu-cp310-cp310-macosx` as well.

**Comment by rusty1s** [comment]
Mh, we followed what PyTorch is doing, e.g., https://download.pytorch.org/whl/torch/. As such, only versions that actually need a prefix have one (Linux/Windows), other's (macOS) don't. I can try to fix this but it is not top prio.

**Reply by quy-ng** [reply]
PyTorch now provides cpu version at https://download.pytorch.org/whl/cpu with following pattern. 

<img width="339" alt="image" src="https://github.com/pyg-team/pytorch_geometric/assets/3946872/7d5746f3-0af5-4807-a72f-84e6cb9a0690">

This works fine with Poetry, pip, and pipenv. Please consider it @rusty1s

**Reply by rusty1s** [reply]
Got it, thanks for the pointer. I will likely update this once we release a new version with PyTorch 2.1.0. Would that be sufficient?

**Reply by quy-ng** [reply]
Hi @rusty1s, is there any chance Torch Geometry will address this conversation?

**Reply by rusty1s** [reply]
Sorry for the delay. We haven't followed up with this yet :(
